bee-snapshot/src/global/file.rs

```diff
@@ -46,12 +47,12 @@ impl GlobalSnapshot {
 
         for line in reader.lines() {
             let line = line.map_err(|_| Error::FailedIO)?;
-            let tokens = line.split(";").collect::<Vec<&str>>();
+            let tokens = line.split(';').collect::<Vec<&str>>();
 
-            if tokens.len() < 2 {
-                return Err(Error::MissingSemicolon);
-            } else if tokens.len() > 2 {
-                return Err(Error::ExtraneousSemicolon);
+            match tokens.len().cmp(&2) {
+                Ordering::Less => return Err(Error::MissingSemicolon),
+                Ordering::Greater => return Err(Error::ExtraneousSemicolon),
+                Ordering::Equal => (),
```

Comment:
Hmmm not sure about this one, less readable IMO

Reply:
Please don't be mad but I prefer the previous version :(

Reply:
I'm so mad I'm going to change it back to the previous version :trollface: